qa/vagrant/build.gradle

```diff
@@ -39,3 +54,18 @@ setupPackagingTest {
     expectedPlugins.setText(plugins.join('\n'), 'UTF-8')
   }
 }
+
+esvagrant {
+  testMainClass 'org.elasticsearch.packaging.PackagingMain'
+}
+
+tasks.test.enabled = false
```

Comment:
Why does this use elasticsearch.build if all these tasks are just disabled? Should this just be a plain groovy project, or is there something else that is needed? If something else, please leave a comment in this file explaining the reasoning.

Reply:
We discussed this - I was able to get most of the precommit tasks passing, now only test and the license checks are disabled